Cargo.toml

```diff
@@ -5,8 +5,8 @@ edition = "2021"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 [lib]
-name = "zarrs"
```

Comment:
I wonder why that worked. In #37, you made it so there are two Rust libraries called `zarrs`, which should be impossible.

Reply:
That's a good point.